tests.ml

```diff
@@ -245,7 +266,22 @@ let suite =
    "scope1ok">:: run_checked (test_scope_1 "c" "c") (has_var "c" (Value.int 0));
    "scope1broken">:: infer_broken_scope (test_scope_1 "a" "c") ["a"];
    "scope1broken2">:: infer_broken_scope (test_scope_1 "a" "b") ["b"; "a"];
-  ]
+   "parser">:: test_parse_disass ("stop\n");
+   "parser1">:: test_parse_disass ("const x = 3\nprint x\nstop\n");
+   "parser2">:: test_parse_disass ("goto l\nx <- 3\nl:\n");
+   "parser3">:: test_parse_disass ("const x = (y + x)\n");
+   "parser4">:: test_parse_disass ("x <- (x == y)\n");
+   "parser5">:: test_parse_disass ("# asdfasdf\n");
+   "parser6">:: test_parse_disass ("branch (x == y) as fd\n");
+   "parser7">:: test_parse_disass ("const x = (y + x)\n x <- (x == y)\n# asdfasdf\nbranch (x == y) as fd\n");
+   "parser8">:: test_parse_disass_file "test.sou";
+   "disass1">:: test_disass_parse (test_sum 10);
+   "disass2">:: test_disass_parse (test_add 1 0);
+   "disass_scope1">:: test_disass_parse test_broken_scope_4;
+   "disass_scope2">:: test_disass_parse test_broken_scope_4_fixed;
+   "disass_scope3">:: test_disass_parse test_broken_scope_5;
+   "parser_scope1">:: test_parse_disass "{a, b} print x\n{a,x,...} #asdf\n";
```

Comment:
yeah, I refrained from this since we still don't know how exactly the invalidate should behave and be used. As soon as we have the first usecases this should stabilize naturally i guess